src/main/navigation/navigation_fixedwing.c

```diff
@@ -463,17 +463,20 @@ int16_t applyFixedWingMinSpeedController(timeUs_t currentTimeUs)
 
 int16_t fixedWingPitchToThrottleCorrection(int16_t pitch)
 {
-    if (pitch > navConfig()->fw.pitch_to_throttle_thresh) {
-        // Positive pitch above threshold
-        return DECIDEGREES_TO_DEGREES(pitch - navConfig()->fw.pitch_to_throttle_thresh) * navConfig()->fw.pitch_to_throttle;
+    // Calculate base throttle correction from pitch moving average
+    const int16_t movingAverageCycles = 128; //Number of main loop cycles for average calculation
```

Comment:
I think the moving average window should be defined in terms of a unit of time instead of loop cycles, so that the amount of smoothing is consistent across loop frequency settings.

Reply:
I see your point about using time for the moving average window. The reason I kept it as a "binary" value is that it is very efficient to just left shift bits for division. Maybe that doesn't matter. @digitalentity What's your thought on this?

Regarding CLI command, I agree. Let me create a CLI variable to keep the current behaviour as default.

Reply:
I have now added `nav_fw_pitch2thr_smoothing` to the cli settings, but kept it as a cycle count for the time being.